src/infrastructure/AccountHttp.ts

```diff
@@ -134,16 +223,40 @@ export class AccountHttp extends Http implements AccountRepository {
                     new UInt64(accountInfoDTO.account.importance),
                     new UInt64(accountInfoDTO.account.importanceHeight),
                 );
+
             });
         }));
     }
 
     public getAccountsNames(addresses: Address[]): Observable<AccountNames[]> {
```

Comment:
:warning: not correct.

you should be copying over the nem2-library/src/api/* classes into the SDK because they are swagger generated.